### PR TITLE
Revert "feat: convert the failure and success scripts to reusable actions"

### PR DESCRIPTION
**Pull request by Hayden-IO**
Reverts sigstore/root-signing#855

### CODE REVIEW

**Comment by Hayden-IO** [comment]
Reverting for now to unblock workflows

File: .github/workflows/scripts/report-success.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+#
+# Copyright 2022 The Sigstore Authors.
```
Comment (bobcallaway) [inline .github/workflows/scripts/report-success.sh:3]: s/2022/2023

File: .github/workflows/scripts/report-failure.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+#
+# Copyright 2022 The Sigstore Authors.
```
Comment (bobcallaway) [inline .github/workflows/scripts/report-failure.sh:3]: s/2022/2023

**Review by bobcallaway — COMMENTED** [review]
LGTM

**Review by bobcallaway — APPROVED** [review]

**Pull request merged**